Resources/Private/JavaScript/media-module/src/components/SideBarRight/Inspector/PropertyInspector.tsx

```diff
@@ -26,9 +27,9 @@ const PropertyInspector = () => {
         approvalAttainmentStrategy: { obtainApprovalToUpdateAsset },
     } = useMediaUi();
     const featureFlags = useRecoilValue(featureFlagsState);
-    const [label, setLabel] = useState<string>(null);
-    const [caption, setCaption] = useState<string>(null);
-    const [copyrightNotice, setCopyrightNotice] = useState<string>(null);
+    const [label, setLabel] = useRecoilState(selectedAssetLabelState);
```

Comment:
Why is it now necessary to have a shared state for the edited asset?

Reply:
If the state is not shared the information in the right sidebar is not updated, when e.g. the title is change in the replace asset dialog while replacing the asset. As far as I understood, the refetch function only updates the image view but not the information in the right sidebar. Or am I wrong?

Reply:
True, it only refreshes when the id of the selected asset changes. Can you then add some comment why this is done this way in this case?